browser/importer/brave_external_process_importer_client.cc

```diff
@@ -14,7 +14,11 @@
 
 namespace {
 bool ShouldUseBraveImporter(importer::ImporterType type) {
-  if (type == importer::TYPE_CHROME)
+  if (type == importer::TYPE_CHROME
+#if BUILDFLAG(IS_WIN)
+      || type == importer::TYPE_EDGE
```

Comment:
This is good - as our import will use the new Chromium type of import (instead of the old Spartan based Edge, which Chrome/Chromium has). We'll want to revisit at some point if Chrome/Chromium ever enhances the Edge import

Reply:
UPDATE: I forgot - Edge is available on macOS and Linux now 😂  So I don't think we want to override. We can change display to `Microsoft Edge Legacy` and keep as-is